forms-flow-theme/scss/customTextArea.scss

```diff
@@ -1,23 +1,44 @@
-$form-textArea-border-radius: var(--radius-0150, 24px);
+$form-textArea-border-radius: var(--radius-0100, 16px);
 $form-textArea-border-color: var(--ff-base-600, #253DF4);
 $form-textArea-focus-outline: 2px solid var(--ff-base-600, #253DF4);
-.custom-textarea {
-    overflow-y: auto;
-    scrollbar-width: none;
-    -ms-overflow-style: none;
-    resize: none; 
-    border-radius: $form-textArea-border-radius !important;
-    border: 1px solid $form-textArea-border-color;
-    padding: 8px 8px 8px 18px !important;
-    font-size: 16px;
-    -webkit-overflow-scrolling: touch; 
-    &:active,
-    &:focus {
-        outline: $form-textArea-focus-outline !important;
-    }
 
-    &::-webkit-scrollbar {
-        display: none;
+.form-input-box{
+    .custom-textarea {
+        overflow-y: auto;
+        scrollbar-width: none;
+        -ms-overflow-style: none;
+        resize: none; 
+        border-radius: $form-textArea-border-radius !important;
+        border: 1px solid $form-textArea-border-color !important;
+        font-size: 16px;
+        -webkit-overflow-scrolling: touch; 
+        flex-grow: 1;
+        &:active,
+        &:focus {
+            outline: $form-textArea-focus-outline !important;
+        }
+    
+        &::-webkit-scrollbar {
+            display: none;
+        }
+    }
+    .input-group-text{
+        height: 100%;
+    }
+    .icon-wrapper {
+        display: flex !important;
+        justify-content: center !important;
+        
+    }
+    .icon-top{
+        align-items: flex-start !important;
+        padding-top: 0.75rem;
```

Comment:
Can we use var(--spacer-075) instead of this ?

Reply:
Updated.